MobileBuy/buy/src/androidTest/java/com/shopify/buy/service/BuyTest.java

```diff
@@ -491,65 +443,55 @@ public void failure(RetrofitError error) {
             }
         });
 
-        latch.await();
```

Comment:
Yes it works, except it ends test suite with terminated state, this I didn't figure out why